resultsdbupdater/consumer.py

```diff
@@ -66,6 +66,8 @@ def _consume_helper(self, msg):
     def consume(self, msg):
         try:
             self._consume_helper(msg)
+        except utils.CreateResultError as e:
+            utils.LOGGER.error(e)
```

Comment:
The following will give you the traceback which might be helpful:
```suggestion
            utils.LOGGER.exception()
```

Reply:
Oh, I don't want the traceback, only the simple text (see `__str__()` method).